docs/usage/configuration/config_documentation.md

```diff
@@ -3355,6 +3355,10 @@ Configuration settings related to push notifications
 This setting defines options for push notifications.
 
 This option has a number of sub-options. They are as follows:
+* `enable_push`: Enables or disables push notification calculation. Note, disabling this will also
+   stop unread counts being calculated for rooms. This mode of operation is intended
+   for homeservers which may only have bots or appservice users connected, or are otherwise
+   not interested in push/unread counters. This is enabled by default.
```

Comment:
```suggestion
   not interested in push/unread counters. Push is enabled by default.
```

The wording doesn't really tell if you are talking about push itself or the option `enable_push`.

Reply:
Sorry my comment is non sense since this is called `enable` and not `disable`...